book.rb

```diff
@@ -0,0 +1,14 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
```

Comment:
- You have done great so far :+1: :100: But there are few requirements you still need to meet before moving to the next milestone. :+1: 

- [ ] As mentioned by the previous reviewer, there needs to be a `Getters and Setters for @rentals`. currently, you only have a `getter` for that. I suggest you update this as required. :+1: :100: Here's what your code could look like. :+1: 

```ruby
attr_accessor :title, :author, :rentals
```